homeassistant/components/knx/schema.py

```diff
@@ -430,3 +430,25 @@ class WeatherSchema:
             vol.Optional(CONF_KNX_HUMIDITY_ADDRESS): cv.string,
         }
     )
+
+
+class FanSchema:
+    """Voluptuous schema for KNX fans."""
+
+    CONF_STATE_ADDRESS = CONF_STATE_ADDRESS
+    CONF_OSCILLATION_ADDRESS = "oscillation_address"
+    CONF_OSCILLATION_STATE_ADDRESS = "oscillation_state_address"
+    CONF_MAX_STEP = "max_step"
+
+    DEFAULT_NAME = "KNX Fan"
+
+    SCHEMA = vol.Schema(
+        {
+            vol.Required(CONF_ADDRESS): cv.string,
+            vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+            vol.Required(CONF_STATE_ADDRESS): cv.string,
```

Comment:
```suggestion
            vol.Optional(CONF_STATE_ADDRESS): cv.string,
```
This should work without state_address as well. Some actuators don't provide one.